addons/portal/static/src/xml/portal_chatter.xml

```diff
@@ -45,7 +45,7 @@
                                         <div class="d-flex px-1">
                                             <button class="o_portal_chatter_attachment_btn btn fa fa-paperclip border-0" type="button" title="Add attachment"/>
                                         </div>
-                                        <button t-attf-data-action="/mail/chatter_post" class="o_portal_chatter_composer_btn btn btn-primary o-last rounded-3" type="submit">Send</button>
```

Comment:
Can you improve the commit message a bit?

- Don't need to use an "arrow" in the commit message
- The commit header should contain all modified modules, so `[IMP] portal[_rating]`
- Would elaborate to explain that we add a way to customize the "Send" button in the portal composer widget by supporting a new option